base/dict.jl

```diff
@@ -449,7 +449,7 @@ Dict{Int64, Int64} with 1 entry:
   2 => 4
 ```
 """
-get!(f::Function, collection, key)
+get!(f::Callable, collection, key)
```

Comment:
```suggestion
get!(f::Union{Function, Type}, collection, key)
```

Reply:
Isn't it just code there? I.e. this change would allow to get the correct docstring when called as e.g. `?get!(Vector, x, y)`, but I don't think using `Callable` here leaks to the user, in the same way we use `Callable` when actually defining the function.